go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/pankona/hashira
 
 require (
+	github.com/adrg/xdg v0.4.0
```

Comment:
It seems this library is not maintained actively. Can you find another package or is it better to implement this by ourself?

Reply:
This library looks used in these few years. And I can understand it is not nessesery to change from the purpose 🤔.